pkg/scheduler/util/utils.go

```diff
@@ -92,30 +91,6 @@ func GetEarliestPodStartTime(victims *extenderv1.Victims) *metav1.Time {
 	return earliestPodStartTime
 }
 
-// SortableList is a list that implements sort.Interface.
```

Comment:
we don't need to create our own "SortableList", we can use [sort.Slice](https://golang.org/pkg/sort/#Slice).

Reply:
I have made the change. Can you review it once?